### DISCUSSION TITLE
оФФ топ

### DISCUSSION THREAD

**Discussion by Kor-Angar** [dosinabox/gothic2_steam_fix#31] [General]
в Steam русскую версию добавили от Акеллы?

![image](https://user-images.githubusercontent.com/49514764/160655478-be94ac2e-aec0-4e99-9c64-d2b1a19f19b7.png)

**Comment by dosinabox** [comment]
Да, прямо по моим рекомендациям и с моими текстурами. И ещё добавили поддержку модов:
https://steamcommunity.com/sharedfiles/filedetails/?id=2786449604

**Comment by dosinabox** [comment]
Так что теперь проект стим-фикса сократится до Юниона, стартера и неофициального обновления.

**Comment by Kor-Angar** [comment]
> Да, прямо по моим рекомендациям и с моими текстурами.

тогда я тебя поздравляю! :-)

**Comment by Kor-Angar** [comment]
> Так что теперь проект стим-фикса сократится до Юниона, стартера и неофициального обновления.

я рад что русская версия стим-фикса будет весить меньше 100 мегабайта :-) 

> теперь проект стим-фикса сократится

но я тебе хочу напомнить что твой стим-фикс ставят так же и на версии Готики не со стима. 
отсылка на твой стим-фикс существует наверное в сотни детских "мануалов", в нормальных инструкциях на всех форумах и  десятках роликах на ютубе. а сейчас русскоязычный сегмент  который используют нестим версию так же вырастит. Это нужно учитывать.

**Comment by Kor-Angar** [comment]
а тут какая то третья позиция появилась? 
![image](https://user-images.githubusercontent.com/49514764/160701905-906f8ab8-23d2-48b2-b57f-f05d248183aa.png)

или так и было?

**Comment by dosinabox** [comment]
workshop это для поддержки модов из стима

**Comment by dosinabox** [comment]
> отсылка на твой стим-фикс существует наверное в сотни детских "мануалов", в нормальных инструкциях на всех форумах и десятках роликах на ютубе

Ну старые версии ведь останутся. Кому надо - возьмут на 600 МБ с озвучкой и установят.

**Comment by Kor-Angar** [comment]
Gothic2_Akella_Covers

[Gothic2_Akella_Covers.zip](https://github.com/dosinabox/g2_steam_fix/files/8406777/Gothic2_Akella_Covers.zip)

**Comment by dosinabox** [comment]
https://steamcommunity.com/sharedfiles/filedetails/?id=2240305342
Это руководство больше не актуально. Все мои исправления английской версии уже вошли в официальную стим-версию, а самый свежий Юнион можно установить из мастерской.

**Comment by Kor-Angar** [comment]
для стим версии неактуально :-)
